assignments/this.js

```diff
@@ -13,14 +16,34 @@
 
 // code example for Window Binding
```

Comment:
function myFunction() {
    console.log(this);
    return name;
}

myFunction('Brett Madrid');